src/web/app/pages-help/instructor-help-page/instructor-help-getting-started/instructor-help-getting-started.component.html

```diff
@@ -4,7 +4,7 @@ <h1 id="top" class="color-orange">Getting Started</h1>
     Welcome to TEAMMATES!
   </p>
   <p>
-    To get started using TEAMMATES, follow the following steps, or watch our <a href="https://www.youtube.com/embed/mDtfmNmRwBM?autoplay=1&rel=0" target="_blank"><i class="fas fa-film"></i> Video Tour</a>.<br>
+    To get started using TEAMMATES, follow the following steps.
```

Comment:
Hi @anuth212, Could you include the `<br>` from before? So that the `For more help...` sentence can be on the next line, as per before? Thanks!

Before:
![](https://user-images.githubusercontent.com/54243224/228148017-a0b2b383-3410-45aa-83dc-3c8e55725bce.png)


Current:
![Screen Shot 2023-04-16 at 3 42 36 PM](https://user-images.githubusercontent.com/25262042/232281043-6b30dfe4-6767-42c6-971b-8123e524f2fc.png)

Reply:
Yup I can do that!